test/framework/src/main/java/org/elasticsearch/bootstrap/BootstrapForTesting.java

```diff
@@ -87,7 +87,7 @@
 
         // check for jar hell
         try {
-            JarHell.checkJarHell();
+            //JarHell.checkJarHell();
```

Comment:
Nothing, it should not have been commited. (I used intellij 2017.1 and it has classpath issues with Gradle)